src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -623,41 +617,22 @@ public override WebResponse GetResponse()
             }
             catch (Exception exception)
             {
-                if (NetEventSource.Log.IsEnabled())
-                {
-                    NetEventSource.Exception(NetEventSource.ComponentType.Web, this, "GetResponse", exception);
-                }
+                NetEventSource.Error(this, exception);
 
                 // if _exception == null, we are about to throw an exception to the user
                 // and we haven't saved the exception, which also means we haven't dealt
                 // with it. So just release the connection and log this for investigation.
                 if (_exception == null)
                 {
-                    if (NetEventSource.Log.IsEnabled())
-                        NetEventSource.PrintWarning(NetEventSource.ComponentType.Web, "Unexpected exception in GetResponse()");
-
-                    if (!ExceptionCheck.IsFatal(exception))
-                    {
-                        if (GlobalLog.IsEnabled)
-                        {
-                            GlobalLog.Assert("Find out why we are getting an unexpected exception.");
-                        }
-
-                        Debug.Fail("Find out why we are getting an unexpected exception.");
-                    }
+                    NetEventSource.Error(this, exception);
```

Comment:
This is one place where I didn't strictly follow what was there previously, because what was there previously was bogus.  This is a valid exception path... in fact, that Ftp test that an issue got opened for around Dns resolution repros for me locally and causes the Debug.Fail that was here to fire.  We should not have asserts fire for anything that's legitimately hittable, so I changed it to the Error that should have been.